components/Dialogs/examples/DialogsAlertViewController.m

```diff
@@ -31,24 +31,24 @@ - (void)collectionView:(UICollectionView *)collectionView didSelectItemAtIndexPa
   [super collectionView:collectionView didSelectItemAtIndexPath:indexPath];
   switch (indexPath.row) {
     case 0:
-      [self didTapShowAlert:nil];
+      [self didTapShowAlert];
       break;
     case 1:
-      [self didTapShowLongAlert:nil];
+      [self didTapShowLongAlert];
       break;
     case 2:
-      [self didTapNondismissingAlert:nil];
+      [self didTapNondismissingAlert];
       break;
     case 3:
-      [self didTapDynamicAlert:nil];
+      [self didTapDynamicAlert];
       break;
     case 4:
-      [self didTapOverpopulatedAlert:nil];
+      [self didTapOverpopulatedAlert];
       break;
   }
 }
 
-- (IBAction)didTapShowAlert:(id)sender {
+- (IBAction)didTapShowAlert{
```

Comment:
super-nit: space after method name, here and below.

Reply:
Done.